frontend/packages/client/src/pages/Community.js

```diff
@@ -256,111 +255,33 @@ export default function Community() {
-                {logo ? (
-                  <div
-                    role="img"
-                    aria-label="community banner"
-                    className={imageClasses}
-                    style={{
-                      width: 85,
-                      height: 85,
-                      backgroundImage: `url(${logo})`,
-                      backgroundRepeat: 'no-repeat',
-                      backgroundPosition: 'center',
-                      backgroundSize: 'cover',
-                    }}
-                  ></div>
-                ) : (
-                  <Blockies
-                    seed={slug ?? `seed-${id}`}
-                    size={10}
-                    scale={9.6}
-                    className="blockies"
-                  />
-                )}
-              </div>
-              <div className="column community-info is-justify-content-space-evenly">
-                <h2 className={titleClassNames}>{community.name}</h2>
-                <p className={memberClassNames}>{totalMembers} members</p>
-                <div className="is-flex">
-                  {members
-                    ? members.slice(0, 6).map(({ addr }, idx) => (
-                        <div
-                          key={`${idx}`}
-                          className="blockies-wrapper is-relative"
-                          style={{ right: `${idx * (notMobile ? 12 : 6)}px` }}
-                        >
-                          <Blockies
-                            seed={addr}
-                            size={notMobile ? 10 : 6}
-                            scale={4}
-                            className="blockies blockies-border"
-                          />
-                        </div>
-                      ))
-                    : null}
-                </div>
-              </div>
-            </div>
-            <JoinCommunityButton
-              communityId={communityId}
-              setTotalMembers={setTotalMembers}
-              onLeaveCommunity={onUserLeaveCommunity}
-              onJoinCommunity={onUserJoinCommunity}
-              classNames="mt-2-mobile"
-            />
-          </div>
-        </div>
-      ) : null}
+      <CommunityHeader
+        bannerImgUrl={community?.bannerImgUrl}
+        id={id}
+        logo={logo}
+        slug={slug}
+        communityName={name}
+        totalMembers={totalMembers}
+        members={members?.slice(0, 6)}
+        joinCommunityButton={
+          <JoinCommunityButton
+            communityId={communityId}
+            setTotalMembers={setTotalMembers}
+            onLeaveCommunity={onUserLeaveCommunity}
+            onJoinCommunity={onUserJoinCommunity}
+            classNames="small-text pb-0-mobile px-0-mobile"
+            extraStyles={notMobile ? { maxWidth: '132px' } : {}}
+            buttonClassNames="py-5 py-4-mobile"
+            borderRadious="rounded-extra-lg"
+            containerAlignment="is-align-items-flex-start"
+          />
+        }
```

Comment:
is `joinCommunityButton` expected to be dynamic? otherwise seems like this component can just live in `CommunityHeader`

Reply:
I did this to avoid passing props for the JoinButton which is using these props: 
```
 communityId={communityId}
 setTotalMembers={setTotalMembers}
 onLeaveCommunity={onUserLeaveCommunity}
 onJoinCommunity={onUserJoinCommunity}
 ```
 but totally fine with passing down this props, what do you think?

Reply:
Ended up moving the component inside the CommunityHeader component